app/services/atmosphere/optimizer.rb

```diff
@@ -14,6 +14,11 @@ def run(hint)
 
     #private
     def satisfy_appliance(appliance)
+
+      action = Action.create(appliance: appliance,
+                             action_type: :satisfy_appliance)
+      action.log("Satisfying appliance started")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.